index.js

```diff
@@ -429,7 +436,11 @@ class Peer extends stream.Duplex {
 
       if (!this._readableState.ended) this.push(null)
       if (!this._writableState.finished) this.end()
-
+      if (typeof window === 'object' && typeof window.chrome === 'object') {
+        if (!forcedGCTimeout) {
+          forcedGCTimeout = setTimeout(forcePeerGC, 5000)
```

Comment:
why 5 seconds? to allow multiple destroy's to be cleaned up at once?

We should also unref the timer

Reply:
honestly, this is a really stupid fix, for a really stupid issue, there are probably way better ways of doing this than what I showed here, which doesn't always work, but yeah, that timeout is to allow multiple peers to be destroyed at once.

Reply:
This is the suggested solution for the mean time, and I believe this is the correct place to handle such "hack".

To not call this so often, I'd even go with 10 seconds.

My only nit requirement: unref the timer upon creation, shouldn't not block process close if it's the last thing standing.

Reply:
unrefed the timeout, but idk about the 10s timeout, 5s is already a lot because in theory it should happen every time a peer is destroyed, at very high amt of peers which are rapidly connecting to the client, 10s might be too long, the client will exceed the wrtc conn limit and essentially kill all future wrtc connections